modin/engines/base/io/text/text_file_reader.py

```diff
@@ -56,218 +57,241 @@ def pathlib_or_pypath(cls, filepath_or_buffer):
 
         return result
 
     @classmethod
     def _read_rows(
         cls,
         f,
-        nrows=None,
-        skiprows=None,
-        quotechar=b'"',
-        is_quoting=True,
-        max_bytes=None,
+        nrows: int,
+        quotechar: bytes = b'"',
+        is_quoting: bool = True,
+        outside_quotes: bool = True,
+        encoding: str = None,
+        newline: bytes = None,
     ):
         """
-        Moves the file offset at the specified amount of rows
-        Note: the difference between `offset` is that `_read_rows` is more
-            specific version of `offset` which is focused of reading **rows**.
-            In common case it's better to use `offset`.
+        Move the file offset at the specified amount of rows.
 
         Parameters
         ----------
-            f: file object
-            nrows: int, number of rows to read. Optional, if not specified will only
-                consider `max_bytes` parameter.
-            skiprows: int, array or callable (optional), specifies rows to skip
-            quotechar: char that indicates quote in a file
-                (optional, by default it's '\"')
-            is_quoting: bool, Whether or not to consider quotes
-                (optional, by default it's `True`)
-            max_bytes: int, Will read new rows while file pointer
-                is less than `max_bytes`. Optional, if not specified will only
-                consider `nrows` parameter, if both not specified will read till
-                the end of the file.
+        f: file object
+        nrows: int
+            Number of rows to read.
+        quotechar: bytes, default b'\"'
+            Indicate quote in a file.
+        is_quoting: bool, default True
+            Whether or not to consider quotes.
+        outside_quotes: bool, default True
+            Whether the file pointer is within quotes or not at the time this function is called.
+        encoding: str, optional
+            Encoding of `f`.
+        newline: bytes, optional
+            Byte or sequence of bytes indicating line endings.
 
         Returns
         -------
-            tuple of bool and int,
-                bool: If file pointer reached the end of the file, but did not find
-                closing quote returns `False`. `True` in any other case.
-                int: Number of rows that was readed.
-        """
-        assert skiprows is None or isinstance(
-            skiprows, int
-        ), f"Skiprows as a {type(skiprows)} is not supported yet."
-
-        if nrows is None and max_bytes is None:
-            max_bytes = float("inf")
+        tuple of bool and int,
+            bool: If file pointer reached the end of the file, but did not find
+            closing quote returns `False`. `True` in any other case.
+            int: Number of rows that was readed.
 
+        Notes
+        -----
+        The difference between `offset` is that `_read_rows` is more
```

Comment:
difference should be between "A" and "B", there can be no difference between "A" without "B" :)
comment does not make much sense